### DISCUSSION TITLE
How to get CreateChatInviteLink?

### DISCUSSION THREAD

**Discussion by heroesf** [rubenlagus/TelegramBots#975] [Q&A] [answered]
CreateChatInviteLink create = new CreateChatInviteLink();
create.setExpireDate(1633826448);
create.setMemberLimit(1);

try {
    execute(create);
} catch (TelegramApiException e) {
    e.printStackTrace();
}

ChatInviteLink inv = create.deserializeResponse(???);
// need: inv .getInviteLink()

I generated a time-limited invitation link.
I can't query what your link is.

How to solve?
I searched all day but couldn't find it.

**Comment by Chase22** [comment, marked as answer]
The [Getting Started](https://github.com/rubenlagus/TelegramBots/wiki/Getting-Started) Guide has everthing you need. deserializeResponse is an internal method and should never be used directly.
After you initialized the CreateChatInviteLink method you need to call the execute method.

**Reply by heroesf** [reply]
thank you, i finally solved it.
